static/js/components/email/EmailCompositionDialog.js

```diff
@@ -43,32 +44,22 @@ export default class EmailCompositionDialog extends React.Component {
     }
   };
 
-  automaticCheckbox = () => {
-    const {
-      activeEmail: { inputs },
-      updateEmailFieldEdit,
-    } = this.props;
-
-    return <Checkbox
-      label="Automatically send this message in the future, whenever new users join who meet these criteria"
-      className="email-automatic"
-      checked={inputs.sendAutomaticEmails || false}
-      value={true}
-      onCheck={e => {
-        updateEmailFieldEdit('sendAutomaticEmails', {
-          target: {
-            value: e.target.checked
-          }
-        });
-      }}
-    />;
-  };
+  renderAutomaticEmailSettings =
+    (sendAutomaticEmails: boolean, updateEmailFieldEdit: () => void): React$Element<*> => (
+    <AutomaticEmailOptions
+      automaticEmailType={sendAutomaticEmails ? EMAIL_CAMPAIGN : ONE_TIME_EMAIL}
+      setAutomaticEmailType={updateEmailFieldEdit} />
```

Comment:
I think it would be best here to do something like what we had before, i.e. just pass down an `onChange` function that can be used in the `AutomaticEmailOptions` component without too much fuss.